src/core/a-animation.js

```diff
@@ -373,6 +379,19 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   };
 
   /**
+   * Match the scema type to color
+   * @return {bool} if the schema is of type color
+   */
+  function isColor () {
+    var componentName = attributeSplit[0];
+    var propertyName = attributeSplit[1];
+    var component = el.components[componentName];
+    var schema = component && component.schema;
+    if (schema) { return schema[propertyName].type === 'color'; }
```

Comment:
@dmarcos Yea I approached it from the idea that it would tween values from your passed object and whatever it's value already was.  As for type checking it was a pure number tween so it would just diff the values calculate delta from last from and decide the amount to tween based on the current framerate.

@ngokevin Understood on tween.js.  I ended up writing that because it didn't take much code to accomplish tween js w/o the overhead of another library.  

I am still not positive this would even work on 'material.specular' as we discussed earlier.  I need to dig in a bit more to how to setup a color value for specular and then try to see if we can animate that as well.